include/swift/AST/DiagnosticsSema.def

```diff
@@ -1973,7 +1973,7 @@ WARNING(protocol_usable_from_inline_warn,none,
         "protocol %select{refined|used}0 by '@usableFromInline' protocol "
         "should be '@usableForInline' or public", (bool))
 ERROR(protocol_property_must_be_computed_var,none,
-      "immutable property requirement must be declared as 'var' with a "
+      "protocols cannot require properties to be immutable; declare read-only properties by using 'var' with a "
```

Comment:
It doesn’t look like this is within 80 columns, could you please fix that?

Reply:
Sorry, I didn't notice that. I fixed it now.